plugins/woocommerce-blocks/bin/webpack-configs.js

```diff
@@ -378,20 +383,17 @@ const getFrontConfig = ( options = {} ) => {
 				minSize: 200000,
 				automaticNameDelimiter: '--',
 				cacheGroups: {
-					...getCacheGroups(),
-					'base-components': {
-						test: /\/assets\/js\/base\/components\//,
-						name( module, chunks, cacheGroupKey ) {
-							const moduleFileName = module
-								.identifier()
-								.split( '/' )
-								.reduceRight( ( item ) => item );
-							const allChunksNames = chunks
-								.map( ( item ) => item.name )
-								.join( '~' );
-							return `${ cacheGroupKey }-${ allChunksNames }-${ moduleFileName }`;
+					commons: {
+						test: /[\\/]node_modules[\\/]/,
+						name: 'wc-blocks-vendors',
+						chunks: ( chunk ) => {
+							return (
+								chunk.name !== 'product-button-interactivity'
```

Comment:
For some reason, turning on vendor bundling in the `product-button-interactivity` bundle caused some APIs of that bundle to be tree-shaken. I've opted just not to vendor bundle for that at the moment. It was the only place there was an issue. The chunk is loaded in the shop page anyway so won't have an impact on cart and checkout. Also there will be some refactoring to remove the dependency on `@wordpress/components` that this has (due to using the woo notice component), when that comes up i'll revisit if this one can (or needs) to be further optimized.